package.json

```diff
@@ -46,16 +47,22 @@
     "@babel/eslint-parser": "^7",
     "@brightspace-ui/stylelint-config": "^0.7",
     "@open-wc/testing": "^3",
+    "@rollup/plugin-dynamic-import-vars": "^2",
+    "@rollup/plugin-node-resolve": "^15",
     "@web/dev-server": "^0.1",
     "@web/test-runner": "^0.15",
     "@web/test-runner-playwright": "^0.9",
     "axe-core": "^4",
     "chalk": "^5",
     "eslint": "^8",
     "eslint-config-brightspace": "^0.19",
+    "glob-all": "^3",
     "lit-analyzer": "^1",
     "messageformat-validator": "^2",
     "node-sass": "^7",
+    "rollup": "^2",
+    "rollup-plugin-copy": "^3",
+    "rollup-plugin-delete": "^2",
```

Comment:
Kind of unfortunate that we need to install these in dev and in all the other jobs that _don't_ need them... can't think of a great alternative though other than manually installing them just in the eventual workflow that runs this.

Reply:
Yeah... `rollup` and `@rollup/plugin-node-resolve` were already being indirectly included, though `@rollup/plugin-node-resolve` now ends up being installed twice with different versions (v13 and v15).